onbuild/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM jusbrasil/python-web:0.5
+FROM jusbrasil/python-web:0.4.1
```

Comment:
Why are you downgrading version?

Reply:
The 0.5 version was my previous attempt, which apparently broke webpy. The current image in production is 0.4, so I'm creating 0.4.1 now

Reply:
In the future someone could bump this to 0.5 and the push/build will break.
It's better to bump to 0.5.1, unless you delete 0.5 from Docker registry.

Reply:
I prefer to delete the 0.5 from the registry, to make sure no one tries to use it until we sort this out.

Reply:
Done, these tags are no longer in our registry